eo-runtime/src/test/java/EOorg/EOeolang/EOcageTest.java

```diff
@@ -230,7 +229,7 @@ void doesNotThrowIfDataizesConcurrently() {
             );
             final int threads = 500;
             MatcherAssert.assertThat(
-                AtCompositeTest.TO_ADD_MESSAGE,
+                "Dataizes concurrent threads failed",
```

Comment:
@ZacParize I think you should follow the same format you used in the messages above. When we "assert" something, a message should say that we expected something to be done, but it wasn't for some reason

Reply:
Ok, got it.